common/metrics/metricstest/metricstest.go

```diff
@@ -55,29 +57,59 @@ type (
 		sampleValue float64
 	}
 
+	histogramBucket struct {
```

Comment:
open question: Should this be uppercased? So other packages can use this struct when running histogram tests?

Reply:
b/c lint complained, answer is yes